src/main/kotlin/lotto/LottoTicket.kt

```diff
@@ -0,0 +1,17 @@
+package lotto
+
+class LottoTicket(numbers: List<Int>) {
+    val numbers: List<Int> = numbers.sorted()
+
+    init {
+        require(numbers.size == 6) {
+            "로또 번호는 총 6개여야 합니다"
+        }
+        require(numbers.all { (1..45).contains(it) }) {
+            "로또 번호는 1에서 45까지만 허용합니다"
```

Comment:
코틀린스럽게 코드를 짠다면 in 키워드를 활용해볼 수도 있어요 :)
```suggestion
        require(numbers.all { it in 1..45 }) {
            "로또 번호는 1에서 45까지만 허용합니다"
```